website/docs/faqs/Core/install-python-compatibility.md

```diff
@@ -5,9 +5,15 @@ sidebar_label: 'Python version'
 id: install-python-compatibility
 ---
 
-<VersionBlock firstVersion="1.1">
+<VersionBlock firstVersion="1.4">
 
-The latest version of `dbt-core` is compatible with Python versions 3.7, 3.8, 3.9, and 3.10.
+The latest version of `dbt-core` is compatible with Python versions 3.7, 3.8, 3.9, 3.10, and 3.11.
```

Comment:
Let's include an additional note about 3.11 here for now, something to this effect:

> Full support for the newest versions of Python depends on which adapter you're using, since adapter plugins have additional third-party dependencies. As of January 2023, `dbt-snowflake` does not yet support Python 3.11 ([relevant issue in `snowflake-connector-python`](https://github.com/snowflakedb/snowflake-connector-python/issues/1345)).

(If we do include the date, we'd need a way to remind ourselves to update this later—perhaps by opening an issue?)

Let's also add a note to the v1.4 migration guide (pending https://github.com/dbt-labs/docs.getdbt.com/pull/2646):

> ### Support for Python 3.11
> 
> Python 3.11 was released in October 2022. It is officially supported in `dbt-core` v1.4, though full support depends also on your data platform adapter plugin.
> 
> According to [the Python maintainers](https://docs.python.org/3/whatsnew/3.11.html), "Python 3.11 is between 10-60% faster than Python 3.10." We encourage you to try a `dbt parse` with dbt Core v1.4 + Python 3.11, and compare it to the result of `dbt parse` on dbt Core v1.3 + Python 3.10. If you notice a significant speedup, let us know!

Reply:
Added notes to both this page and the update guide. Will open an issue to remind me to remove the date.